<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>883</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="sol">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>590</y>
      <width>261</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Pictures/solesote.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="pla4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>venus pero este es marte.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="pla6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>saturno.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="pla5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Jupiter.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="pla8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>neptuno.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="pla3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Pictures/tierra.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="pla2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>marte.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="pla7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Urano.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="pla1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>mercurio.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="botonIniciar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>780</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Iniciar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>740</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Mercurio</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>630</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>venus</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Tierra</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Marte</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Jupiter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Saturno</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Urano</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Neptuno</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>511</width>
      <height>841</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>20</width>
      <height>691</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ciclo planetario</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_neptuno">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed Extra Bold</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>uy quieto</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_urano">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed Extra Bold</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>uy quieto</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_saturno">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>240</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed Extra Bold</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>uy quieto</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_jupiter">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>320</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed Extra Bold</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>uy quieto</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_marte">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>400</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed Extra Bold</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>uy quieto</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tierra">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>490</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed Extra Bold</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>uy quieto</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_venus">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>580</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed Extra Bold</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>uy quieto</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_mercurio">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>680</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed Extra Bold</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>uy quieto</string>
    </property>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>sol</zorder>
   <zorder>pla4</zorder>
   <zorder>pla6</zorder>
   <zorder>pla5</zorder>
   <zorder>pla8</zorder>
   <zorder>pla3</zorder>
   <zorder>pla2</zorder>
   <zorder>pla7</zorder>
   <zorder>pla1</zorder>
   <zorder>botonIniciar</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>line</zorder>
   <zorder>label_9</zorder>
   <zorder>label_neptuno</zorder>
   <zorder>label_urano</zorder>
   <zorder>label_saturno</zorder>
   <zorder>label_jupiter</zorder>
   <zorder>label_marte</zorder>
   <zorder>label_tierra</zorder>
   <zorder>label_venus</zorder>
   <zorder>label_mercurio</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>513</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPLANETAS">
    <property name="title">
     <string>PLANETAS</string>
    </property>
   </widget>
   <addaction name="menuPLANETAS"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
